iris/meta.yaml

```diff
@@ -1,23 +1,22 @@
+{% set version = "1.11.0" %}
+
 package:
     name: iris
-    version: 1.10.0
+    version: {{ version }} 
 
 source:
     git_url: https://github.com/SciTools/iris.git
-    git_tag: v1.10.0
+    git_tag: v{{ version }}
 
 build:
-    number: 2
-    # See https://github.com/SciTools/cartopy/pull/676
-    skip: True  # [py35]
+    number: 0
 
 requirements:
     build:
-        - python  # [not osx]
-        - python >=2.7,<3  # [osx]
```

Comment:
This here change does seem to be the nub of the problems! I wonder if we should reinstate this line so that we can at least get *some* OSX builds.